src/Symfony/Component/HttpFoundation/HeaderUtils.php

```diff
@@ -143,6 +146,54 @@ public static function unquote(string $s): string
         return preg_replace('/\\\\(.)|"/', '$1', $s);
     }
 
+    /**
+     * Generates a HTTP Content-Disposition field-value.
+     *
+     * @param string $disposition      One of "inline" or "attachment"
+     * @param string $filename         A unicode string
+     * @param string $filenameFallback A string containing only ASCII characters that
+     *                                 is semantically equivalent to $filename. If the filename is already ASCII,
+     *                                 it can be omitted, or just copied from $filename
+     *
+     * @return string A string suitable for use as a Content-Disposition field-value
+     *
+     * @throws \InvalidArgumentException
+     *
+     * @see RFC 6266
+     */
+    public static function makeDisposition($disposition, $filename, $filenameFallback = '')
```

Comment:
why not using `null` as default here?

Reply:
I've just moved the code and don't want to make other changes.

Reply:
Makes sense 👍 
Thank you!

Reply:
IMO new code should have type declarations.

Reply:
done